spec/integration/criteria/where_spec.rb

```diff
@@ -28,7 +28,7 @@
   context 'when using bad arguments' do
     it 'raises' do
       expect { SimpleDocument.where("xxx") }.to raise_error(/Invalid/)
-      expect { SimpleDocument.where(:xxx.eq(1,2)) }.to raise_error(/Invalid/)
```

Comment:
eq(1,2) should still raise an error

Reply:
Nice catch! Of course. I'll change that, thank you!